app/views/spree/admin/orders/bulk_management.html.haml

```diff
@@ -114,13 +141,17 @@
               %a{ :href => '', 'ng-click' => "predicate = 'order.distributor.name'; reverse = !reverse" } Hub
             %th.variant{ 'ng-show' => 'columns.variant.visible' }
               %a{ :href => '', 'ng-click' => "predicate = 'units_variant.full_name'; reverse = !reverse" } Product: Unit
-            %th.quantity{ 'ng-show' => 'columns.quantity.visible' } Quantity
-            %th.max{ 'ng-show' => 'columns.max.visible' } Max
-            %th.final_weight_volume{ 'ng-show' => 'columns.final_weight_volume.visible' } Weight/Volume
-            %th.price{ 'ng-show' => 'columns.price.visible' } Price
+            %th.quantity{ 'ng-show' => 'columns.quantity.visible' }
+              = t "products_quantity"
+            %th.max{ 'ng-show' => 'columns.max.visible' }
+              = t "shop_variant_quantity_max"
+            %th.final_weight_volume{ 'ng-show' => 'columns.final_weight_volume.visible' }
+              = t "weight_volume"
+            %th.price{ 'ng-show' => 'columns.price.visible' }
+              = t "products_price"
             %th.actions
             %th.actions
-              Ask?&nbsp;
+              = t "ask"
```

Comment:
= t "ask"&nbsp;
I kept t "#name#" because in this file all translation have been done like that....